keyboards/planck/keymaps/vifon/keymap.c

```diff
@@ -89,28 +91,46 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
     {_______, _______, KC_LGUI, KC_LALT, _______, _______,     _______,        _______, _______, _______, _______, _______}
 },
 [_DYN]= { /* special */
-    {_______,  DYN_REC_START1, DYN_MACRO_PLAY1, _______, _______, _______, _______, KC_APP,  KC_INS,  _______, KC_PSCR, KC_PAUS},
-    {_______,  DYN_REC_START2, DYN_MACRO_PLAY2, _______, _______, _______, _______, _______, _______, KC_CAPS, KC_SLCK, KC_NLCK},
-    {KM_SHLK,  _______,        _______,         _______, _______, _______, _______, _______, _______, _______, _______, _______},
-    {_______,  _______,        _______,         _______, _______, _______, _______, _______, _______, _______, _______, _______}
+    {KM_HOLD,  DYN_REC_START1, DYN_MACRO_PLAY1, _______, _______,    _______, _______, KC_APP,  KC_INS,  _______, KC_PSCR, KC_PAUS},
+    {KC_LOCK,  DYN_REC_START2, DYN_MACRO_PLAY2, _______, _______,    _______, _______, _______, _______, KC_CAPS, KC_SLCK, KC_NLCK},
+    {KM_SHLK,  _______,        _______,         _______, _______,    _______, _______, _______, _______, _______, _______, _______},
+    {_______,  _______,        _______,         _______, KM_PP_GAME, DF(_QW), DF(_QW), DF(_PP), _______, _______, _______, _______}
 },
 };
 
 #undef _______
+#undef XXXXXXX
 
 const uint16_t PROGMEM fn_actions[] = {
     ACTION_MODS_TAP_KEY(MOD_RSFT, KC_ENT),
+    ACTION_MODS_TAP_KEY(MOD_RSFT, KC_DOT),
```

Comment:
These all could be replaced with `MT(mod, kc)`.
https://docs.qmk.fm/#/feature_advanced_keycodes?id=mod-tap

No need to use the fn_actions block here, especially as it's depreciated.

Reply:
Oh, great! I had the first action for over two years when `MT()` wasn't available I believe, only `SFT_T()` which only had the left shift modifier. And now I just added two analogous actions. I'm going to upgrade it in a few minutes.

Reply:
Well, SFT_T is an alias for MT, actually. 

And there is `RSFT_T` and `RCTL_T`, too.  

And yeah, all of the "functions" have an alias now, so, it's much better now. :)